integreat_cms/cms/views/organizations/organization_list_view.py

```diff
@@ -0,0 +1,47 @@
+import logging
+
+from django.conf import settings
+from django.contrib import messages
+from django.core.paginator import Paginator
+from django.shortcuts import render, redirect
+from django.utils.decorators import method_decorator
+from django.utils.translation import ugettext as _
+from django.views.generic import TemplateView
+from django.views import View
+from django.http import HttpResponse
+from django.views.generic.base import ContextMixin
+
+
+logger = logging.getLogger(__name__)
+
+
+class OrganizationListView(TemplateView):
+    template_name = "organizations/organization_list.html"
+    extra_context = {
+        "current_menu_item": "organizations",
+    }
+
+    def get(self, request, *args, **kwargs):
+        region = request.region
+        if not request.user.has_perm("cms.change_organization"):
+            messages.warning(
+                request,
+                ("You don't have the permission to edit or create organizations."),
```

Comment:
```suggestion
                _("You don't have the permission to edit or create organizations."),
```
We want to translate all hard-coded strings

Reply:
Missed that one :)